30daysofscala/src/main/scala/example/Assignment5.scala

```diff
@@ -0,0 +1,35 @@
+object Point {
+
+  def stringToInt(x: String): Option[Int] = {
+    try {
+      Some(x.toInt)
+    } catch {
+      case e: Exception => None
+    }
+  }
+
+  def apply(sx: String, sy: String): Point = {
+    val x = stringToInt(sx)
+    val y = stringToInt(sy)
+    (x, y) match {
+
+      case (Some(x), Some(y)) => Point(x, y)
+      case (_, _)             => Point(0, 0)
+    }
+  }
+
+}
+case class Point(x: Int, y: Int)
+
+sealed trait Lang
+case object Scala extends Lang
+case object Java extends Lang
+case object Python extends Lang
+
+object SearchIfObjectExtendLang extends App {
+  def convertToString(lan: Lang): String = lan match {
+
+    case Scala | Java | Python => s"Lang language is good"
+    case _                     => "Not within the language Lang"
+  }
```

Comment:
`object SearchIfObjectExtendLang extends App {
  def convertToString(lan: Lang): String = lan match {
    case Scala | Java | Python => s"$lan language is good"
    case _                     => "Not within the language Lang"
  }
  println(convertToString(Python))
}`

Reply:
Updated your code, so that it will do a printout based on the selected case object